package.json

```diff
@@ -107,6 +107,7 @@
     "async-disk-cache": "^2.1.0",
     "bent": "^7.3.12",
     "https-proxy-agent": "^4.0.0",
+    "moment": "^2.29.1",
```

Comment:
From [the moment readme](https://momentjs.com/docs/#/use-it/): We recognize that many existing projects may continue to use Moment, but we would like to discourage Moment from being used in new projects going forward. Instead, we would like to [recommend alternatives](https://momentjs.com/docs/#/-project-status/recommendations/) that are excellent choices for use in modern applications today.

Can we use day.js or some other zero dependency library with a note to drop the dependency once Temporal becomes part of the JS standard?

Reply:
Thanks! I didn't notice this readme, I was thinking moment.js was somehow widely used library which has a C# `TimeSpan` equivalent in JS. I think I'd better just revert to use int64 for the time duration.

Reply:
updated